orca/orca_test.go

```diff
@@ -40,7 +40,7 @@ func TestToLoadReport(t *testing.T) {
 		{
 			name:    "no load report in metadata",
 			md:      metadata.MD{},
-			wantErr: true,
+			wantErr: false,
 		},
 		{
```

Comment:
This is where I think the unit tests I mention should go. I think these cases are important since this function gets invoked so much when ORCA is configured.

Reply:
Done